packages/core-mobile/app/store/notifications/listeners/listeners.ts

```diff
@@ -90,7 +93,7 @@ export const addNotificationsListeners = (
       setAccount,
       onFcmTokenChange,
       turnOnNotificationsFor,
```

Comment:
I think we should do a matcher/predicate here for turnOnNotificationsFor === balance change channel id